src/redux/books/books.js

```diff
@@ -1,30 +1,42 @@
-const initialState = [];
+const books = [{
+  author: 'Frank Herbert',
+  title: 'Dune',
+  id: 0,
+},
+{
+  author: 'Berhanu Tarekegn',
+  title: 'Time mach',
+  id: 2,
+},
+
+{
+  author: 'Girma Trekegn',
+  title: 'Money',
+  id: 3,
```

Comment:
- [optional] nice work. dont you think it will better if you reduce the books in the array to one so the user can add his/her books manually.